src/FluentMigrator.Runner.Oracle/Generators/Oracle/OracleQuoterBase.cs

```diff
@@ -15,13 +15,40 @@
 #endregion
 
 using System;
+using System.Collections.Generic;
+using System.Linq;
 
 using FluentMigrator.Runner.Generators.Generic;
 
 namespace FluentMigrator.Runner.Generators.Oracle
 {
     public class OracleQuoterBase : GenericQuoter
     {
+        private const int MaxStringLength = 3900;
```

Comment:
Consider adding a comment explaining where 3900 comes from.

Reply:
@PhenX Actually, I'm wondering why this value isn't 4000, based on our discussion on this issue and comments from StackOverflow indicating 4000?

Reply:
Yep, that's 4000, I chose to keep a margin of 100 for multibyte characters 😄

Reply:
But I think I might have made a mistake : I can't seem to find any report of the error "PLS-00172: string literal too long" for NCLOB columns, I'll have to check, I'm currently making a Docker image with Oracle 12C.
By the way, is everything OK at your side ? In France, it could be worse ...

Reply:
Following up from other PR comment: Let's rename `MaxStringLength` to `MaxStringLengthInBytes`. When I first read this, after not thinking about the problem for a month, I forgot the issue is related to byte length, not character length.  My brain for some reason just assumed we were only thinking about characters here.  My comments about string splitting were actually split-brain: I was thinking about splitting BMP characters by accident, not thinking about the definition of 4000 being bytes.

Reply:
> By the way, is everything OK at your side ? In France, it could be worse ...

I have been WFH from end of February / early March (had to make two trips into the office).  I live in the suburbs so I am much less at risk for community spread disease.  But it sucks not being able to see friends.  I am through the hard part of adjusting.  I bought Scrabble this week to play with my wife so we don't lose our minds.  If you have any suggestions on how to keep sane, you can find me on gitter.im in the [FluentMigrator chat](https://gitter.im/FluentMigrator/fluentmigrator) :)

Reply:
@PhenX One last thing - I updated #1129 to include a link to the semi-official Oracle 11xe docker image, if that makes your testing easier.  (I don't really understand g, c, i suffixes on Oracle releases but I imagine if you get something working on 11xe, it's likely to work on 19xe)